hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/HollowAccountFinalizationSuite.java

```diff
@@ -734,12 +738,76 @@ private HapiSpec completedHollowAccountsTransfer() {
                 }));
     }
 
-    private HapiCryptoTransfer sendToEvmAddressFromECDSAKey(final HapiSpec spec, final String key) {
+    private HapiSpec txnWith2CompletionsAndAnother2PrecedingChildRecords() {
```

Comment:
Add a  test similar to this except with 2 child record creating transactions and verify that only the preceding child records were created and no child records for the failed child logic.